stream-chat-android-ui-components-sample/src/main/res/layout/channels_empty_view.xml

```diff
@@ -0,0 +1,78 @@
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/emptyContainer"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    >
+
+    <ImageView
+        android:id="@+id/emptyImage"
+        android:layout_width="112dp"
+        android:layout_height="112dp"
+        android:src="@drawable/empty_chats"
+        app:layout_constraintBottom_toTopOf="@+id/emptyTitle"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintVertical_chainStyle="packed"
+        tools:ignore="ContentDescription"
+        />
+
+    <TextView
+        android:id="@+id/emptyTitle"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="48dp"
+        android:layout_marginTop="32dp"
+        android:layout_marginEnd="48dp"
+        android:gravity="center"
+        android:text="@string/channel_list_empty_title"
+        android:textColor="@color/stream_ui_text_color_strong"
+        android:textSize="@dimen/stream_ui_text_large"
+        android:textStyle="bold"
+        app:layout_constrainedWidth="true"
+        app:layout_constraintBottom_toTopOf="@+id/emptyDescription"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/emptyImage"
+        />
+
+    <TextView
+        android:id="@+id/emptyDescription"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="48dp"
+        android:layout_marginTop="8dp"
+        android:layout_marginEnd="48dp"
+        android:gravity="center"
+        android:text="@string/channel_list_empty_description"
+        android:textColor="@color/stream_ui_text_color_strong"
```

Comment:
by checking figma, I believe this is `stream_ui_text_color_light` instead of `stream_ui_text_color_strong`

Reply:
Good catch, thank you 👍